app/package/models.py

```diff
@@ -122,16 +122,14 @@ def build_s3_object_url(self, domain_name, path):
                    key=self.build_s3_key(path))
 
     def generate_pre_signed_post_object(self, path, md5,
-                                        acl='public-read',
-                                        file_type='binary/octet-stream'):
+                                        acl='public-read'):
         """
         This method produce required data to upload file from client side
         for uploading data at client side
         :param path: The relative path of the object
         :param md5: The md5 hash of the file to be uploaded
 
         :param acl: The object ACL default is public_read
-        :param file_type: The type of the file, default is binary/octet-stream
```

Comment:
Agree on simple removal but think you need to add something to docstring to indicate that file_type will always be set to text/plain and why so future devs know why we did this (and don't refer to the issue - just write it out briefly here).